src/sonic-yang-mgmt/tests/libyang-python-tests/sonic_config_data.json

```diff
@@ -0,0 +1,276 @@
+{
+	"test-sonic-vlan:sonic-vlan": {
+		"test-sonic-vlan:VLAN_INTERFACE": {
+			"VLAN_INTERFACE_LIST": [{
+					"vlanid": 111,
+					"ip-prefix": "2000:f500:45:6709::1/64",
+					"scope": "global",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 111,
+					"ip-prefix": "10.1.1.65/26",
+					"scope": "global",
+					"family": "IPv4"
+				},
+				{
+					"vlanid": 111,
+					"ip-prefix": "fe80::1/10",
+					"scope": "local",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "2000:f500:41:4e9::1/64",
+					"scope": "global",
+					"family": "IPv6"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "10.1.5.65/26",
+					"scope": "global",
+					"family": "IPv4"
+				},
+				{
+					"vlanid": 555,
+					"ip-prefix": "fe80::1/10",
+					"scope": "local",
+					"family": "IPv6"
+				}
+			]
+		},
+
+		"test-sonic-vlan:VLAN": {
+			"VLAN_LIST": [{
+					"vlanid": 111,
+					"description": "server_vlan",
+					"dhcp_servers": [
+						"10.1.7.116"
+					],
+					"mtu": "9216",
+					"admin_status": "up",
+					"members": [
+						"Ethernet8",
+						"Ethernet3",
+						"Ethernet0",
+						"Ethernet1",
+						"Ethernet6",
+						"Ethernet4",
+						"Ethernet5",
+						"Ethernet9",
+						"Ethernet2"
```

Comment:
why this order, can you sort?

Reply:
Again this is test data for sonic_yang_mgmt library, since we do not have any constraint in test yang models for sequence, we can provide the data in any order, and library should be able to process it. I can provide sorted data for readability if needed.

Reply:
Removed this filed, since these are just test yang models.